src/configure/helper/templateHelper.ts

```diff
@@ -93,6 +109,24 @@ const simpleWebAppTemplates: Array<PipelineTemplate> = [
         label: 'Simple application to Windows Web App',
         path: path.join(path.dirname(path.dirname(__dirname)), 'configure/templates/simpleWebApp.yml'),
         language: 'none',
-        targetType: TargetResourceType.WindowsWebApp
+        targetType: TargetResourceType.WebApp,
+        targetKind: WebAppKind.WindowsApp
     }
-];
\ No newline at end of file
+];
+
+const functionTemplates: Array<PipelineTemplate> = [
+    {
+        label: 'Python Function App to Linux Azure Function',
+        path: path.join(path.dirname(path.dirname(__dirname)), 'configure/templates/pythonLinuxFunctionApp.yml'),
+        language: 'python',
```

Comment:
can you also add logic to find if it is a python application and then show this template there. It wouldn't make sense to show this template when we detect the language to be node

Reply:
We show this when we detect it as function app, regardless of node or python.

Reply:
We might need to reconsider this as well. We can maintain the current behaviour I.E. show templates for the lang detected. 
We will have to look back at it with target resource type analysis for Function app, Linux app, Container app and Windows app also.

Reply:
That can't work for function app since we're checking only for root level files. All language related files for function app will be in subfolders